### ISSUE TITLE
setup of version 2024.11.001-beta on windows  10 seem not to work correctly

### ISSUE THREAD

**Issue by sven-binnig** [devonfw/IDEasy#767]
### Expected behavior

As a developer, I want setup the ide for the first time.

### Actual behavior
After downloading and extracting the tar.gz into c:\projects\_ide I have started the setup.bat.
* The .bashrc was altered, but not as described in the ideasy.pdf
* the ide.bat was not copied to the userprofile
* no directory `.ide` was created in the userprofile

When I try to call neary any commandlet (beside shell) I got an error message telling me, that I'm not inside an ide-installation.

### Steps to reproduce (bug) / Use Case of feature request (enhancement)

1. start with a windows-10-pc with an older version of devonfwide (2021-somewhat)
2. as described above

### Affected version:

* OS: Windows 10
* Browser: Edge and Firefox

**Comment by sven-binnig** [comment]
the complete output of the `setup.bat`:
```
C:\projects\_ide>setup
Setting up IDEasy in C:\projects\_ide
Adding C:\projects\_ide\bin to your users system PATH

ERFOLGREICH: Angegebener Wert wurde gespeichert.

ERFOLGREICH: Angegebener Wert wurde gespeichert.
Found bash at C:\Program Files\Git\bin\bash.exe
"C:\Program Files\Git\bin\bash.exe" -l -c "cd \"C:\projects\_ide\";./setup"
Setting up your IDEasy in /c/projects/_ide
Configuring IDEasy in /c/Users/f0280414/.bashrc.
/c/Users/f0280414/.zshrc not found - skipping.
Setup of IDEasy completed
Drücken Sie eine beliebige Taste . . .

C:\projects\_ide>
```

**Comment by jan-vcapgemini** [comment]
@sven-binnig thanks for reporting this issue. It appears to me that the installation worked as expected. The next step would be to create a new project using `ide create yourprojectname` and switching the current working directory to it e.g. `cd yourprojectname` the message `you are not inside an ide-installation` should disappear then. In our next release, this message will be removed as it is misleading. Instead IDEasy will report that the environment variables have been successfully set.

**Comment by hohwille** [comment]
Was more of a documentation bug addressed with PR #776
